src/common-domain/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'java-library'
+    id 'org.springframework.boot'
```

Comment:
Is this plugin needed?

Reply:
yes, it injects spring boot dependencies.